pkg/stanza/fileconsumer/file.go

```diff
@@ -50,7 +50,9 @@ func (m *Manager) Start(persister operator.Persister) error {
 		return fmt.Errorf("read known files from database: %w", err)
 	}
 
-	if len(m.finder.FindFiles()) == 0 {
+	if files, err := m.finder.FindFiles(); err != nil {
+		return fmt.Errorf("find files: %w", err)
+	} else if len(files) == 0 {
```

Comment:
This might be too brittle because any single file that doesn't match the expected regex will result in no files being read.

I think it might be better to handle this situation by only logging a warning when files don't match.